inst/plumber/v1/endpoints.R

```diff
@@ -9,23 +9,23 @@ library(pipapi)
 #* Ensure that version parameter is correct
 #* @filter validate_version
 function(req, res) {
-  # tictoc::tic("filters")
-  # browser()
-  if (!is.null(req$argsQuery$version) & !grepl("swagger", req$PATH_INFO)) {
-    if (!is.null(req$argsQuery$version)) {
-      if (!req$argsQuery$version %in% lkups$versions) {
+
+  #If no arguments are passed, use the latest version
+  if (is.null(req$argsQuery$release_version) && is.null(req$argsQuery$ppp_version) &&
+     is.null(req$argsQuery$version) && is.null(req$argsQuery$identity)) {
+      version <- lkups$latest_release
```

Comment:
I think we should use single `&` here. THe `&&` silently drops extra items if length is > to 1. We should not run in such cases, but I guess it's always better to get a warning.

Reply:
That should not be an issue moving forward. In R, 4.2 length > 1, returns a warning which eventually will start returning error since 4.2.1. 

https://cran.r-project.org/doc/manuals/r-devel/NEWS.html 

>CHANGES IN R 4.2.0
Calling && or || with either argument of length greater than one now gives a warning (which it is intended will become an error).

>CHANGES IN R-devel
Calling && or || with LHS or (if evaluated) RHS of length greater than one is now always an error, with a report of the form

    'length = 4' in coercion to 'logical(1)'

Moreover, `&&` short-circuits the expression and doesn't evaluate other conditions if any of earlier conditions is `FALSE` making it faster than `&`. 

Let me know if you still want me to switch it back to `&` in case if we are not going to use R 4.2 or higher.